openfe/protocols/openmm_rfe/_rfe_utils/relative.py

```diff
@@ -2489,8 +2568,14 @@ def omm_hybrid_topology(self):
         Returns
         -------
         hybrid_topology : simtk.openmm.app.Topology
+
+
+        .. versionchanged:: OpenFE 0.11
```

Comment:
not sure we're tracking changes until 1.0, can probably not bother with these till then

Reply:
I only added this here so we can keep track of the changes we made to HTF - that way when things finally get moved into OpenMMTools, it'll be easier to quickly identify which features changed from perses.